src/components/Popover/Anchor.tsx

```diff
@@ -0,0 +1,10 @@
+import {
+  Anchor as RadixPopoverAnchor,
+  PopoverAnchorProps as RadixPopoverAnchorProps,
+} from "@radix-ui/react-popover";
+
+export type PopoverAnchorProps = RadixPopoverAnchorProps;
```

Comment:
**issue**: please do not expose entire Radix contract, create your own props instead